lib/pos_mempool.go

```diff
@@ -149,6 +154,11 @@ type PosMempool struct {
 	// based off the current state of the mempool and the most n recent blocks.
 	feeEstimator *PoSFeeEstimator
 
+	maxValidationViewConnects uint64
```

Comment:
Here again I wish you had super detailed comments that explained what these control and all of the design decisions around them.